### ISSUE TITLE
✅ 🐞 SVG Icons break the Title layout of Nested Accordions on Firefox and Safari  - [ED-13553]

### ISSUE THREAD

**Issue by jddrums** [elementor/elementor#24925]
### Prerequisites

- [X] I have searched for similar issues in open and closed tickets and cannot find a duplicate.
- [X] I have troubleshooted my issue, and it still exists against the latest stable version of Elementor.

### Description

Regular accordion menu appears like this on an iPhone, NOT the emulator.
![image](https://github.com/elementor/elementor/assets/148893678/cd100700-2381-4536-99e6-c395534559f2)

### Steps to reproduce

Add an Accordian to a page.
Publish
View on a iPhone 12 Safari, or probably any updated iPhone safari.

### Expected behavior

Foir the pluses to be aligned to the left in that case.  If you play around with the settings, its never right.

### Isolating the problem

- [X] This bug happens when only the Elementor (and Elementor Pro) plugins are active.
- [X] This bug happens with the Hello Elementor theme active.
- [X] I can reproduce this bug consistently by following the steps I described above.

### Elementor System Info

<details>
<summary>Click to reveal</summary>

````txt


== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: Percona Server (GPL), Release 25, Revision 9468fd1d v8.0.33-25
	PHP Version: 8.0.30
	PHP Memory Limit: 256M
	PHP Max Input Vars: 10000
	PHP Max Post Size: 100M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.4.2
	Site URL: https://94p.cd2.myftpupload.com
	Home URL: https://94p.cd2.myftpupload.com
	WP Multisite: No
	Max Upload Size: 100 MB
	Memory limit: 40M
	Max Memory limit: 256M
	Permalink Structure: /%year%/%monthnum%/%postname%/
	Language: en-US
	Timezone: 0
	Debug Mode: Inactive

== Theme ==
	Name: Neresa WP
	Version: 1.1
	Author: CocoBasic
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en_US
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/120.0.0.0 Safari/537.36 Edg/120.0.0.0

== Active Plugins ==
	Advanced Google reCAPTCHA
		Version: 1.18
		Author: WebFactory Ltd

	Check &amp; Log Email
		Version: 1.0.8
		Author: WPOmnia

	CoBlocks
		Version: 3.1.5
		Author: GoDaddy

	CocoBasic - Neresa Elementor Widgets
		Version: 1.1
		Author: CocoBasic

	CocoBasic - Neresa WP
		Version: 1.0
		Author: CocoBasic

	Contact Form 7
		Version: 5.8.6
		Author: Takayuki Miyoshi

	Elementor
		Version: 3.19.0-dev6
		Author: Elementor.com

	Elementor Beta (Developer Edition)
		Version: 1.1.2
		Author: Elementor.com

	Elementor Header &amp; Footer Builder
		Version: 1.6.23
		Author: Brainstorm Force, Nikhil Chavan

	Rank Math SEO
		Version: 1.0.211
		Author: Rank Math

	Redirection
		Version: 5.4.1
		Author: John Godley

	Site Kit by Google
		Version: 1.118.0
		Author: Google


== Must-Use Plugins ==
	Object Cache Pro (MU)
		Version: 1.18.2
		Author: Rhubarb Group

	System Plugin
		Version: 4.106.0
		Author: 


== Elementor Experiments ==
	Improved Asset Loading: Active
	Improved CSS Loading: Active
	Inline Font Icons: Active
	Additional Custom Breakpoints: Active
	admin_menu_rearrangement: Inactive by default
	Flexbox Container: Active
	Upgrade Swiper Library: Inactive
	Grid Container: Active
	Editor Top Bar: Active
	Optimized Gutenberg Loading: Active by default
	Build with AI: Active by default
	Landing Pages: Active
	Nested Elements: Active
	Lazy Load Background Images: Active
	Optimize Image Loading: Active by default
	Pages Panel: Inactive by default
	Global Style Guide: Active


== Log ==
	
Log: showing 20 of 372024-01-15 20:13:45 [info] elementor::elementor_updater Started 
2024-01-15 20:13:45 [info] Elementor/Upgrades - _on_each_version Start  
2024-01-15 20:13:45 [info] Elementor/Upgrades - _on_each_version Finished 
2024-01-15 20:13:45 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.19.0-dev2',
  'to' => '3.19.0-dev3',
)]
2024-01-15 20:13:45 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.19.0-dev2',
  'to' => '3.19.0-dev3',
)]
2024-01-17 18:49:57 [info] elementor::elementor_updater Started 
2024-01-17 18:49:57 [info] Elementor/Upgrades - _on_each_version Start  
2024-01-17 18:49:57 [info] Elementor/Upgrades - _on_each_version Finished 
2024-01-17 18:49:57 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.19.0-dev3',
  'to' => '3.19.0-dev4',
)]
2024-01-17 18:49:57 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.19.0-dev3',
  'to' => '3.19.0-dev4',
)]
2024-01-22 13:31:09 [info] elementor::elementor_updater Started 
2024-01-22 13:31:09 [info] Elementor/Upgrades - _on_each_version Start  
2024-01-22 13:31:09 [info] Elementor/Upgrades - _on_each_version Finished 
2024-01-22 13:31:09 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.19.0-dev4',
  'to' => '3.19.0-dev5',
)]
2024-01-22 13:31:09 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.19.0-dev4',
  'to' => '3.19.0-dev5',
)]
2024-01-24 13:27:34 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.19.0-dev5',
  'to' => '3.19.0-dev6',
)]
2024-01-24 13:27:34 [info] elementor::elementor_updater Started 
2024-01-24 13:27:34 [info] Elementor/Upgrades - _on_each_version Start  
2024-01-24 13:27:34 [info] Elementor/Upgrades - _on_each_version Finished 
2024-01-24 13:27:34 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.19.0-dev5',
  'to' => '3.19.0-dev6',
)]

PHP: showing 2 of 2PHP: 2023-12-10 12:18:04 [warning X 9][../wp-content/plugins/elementor/modules/history/revisions-manager.php::363] Undefined array key &quot;editor_post_id&quot; [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2024-01-11 13:05:03 [warning X 2][../wp-content/plugins/elementor/includes/managers/image.php::113] Trying to access array offset on value of type bool [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]

JS: showing 4 of 4JS: 2023-12-10 12:26:13 [error X 10]../wp-content/plugins/elementor/assets/js/4fdaa70e951ad90db2f2.bundle.min.js:2:7786] Cannot read properties of null (reading &#039;dataset&#039;) 
JS: 2024-01-22 13:04:46 [error X 1]../wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.19.0-dev4:3:323770] Cannot read properties of undefined (reading &#039;html&#039;) 
JS: 2024-01-27 00:22:27 [error X 1]../wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.19.0-dev6:3:647968] Cannot read properties of undefined (reading &#039;isDesignable&#039;) 
JS: 2024-01-27 00:59:31 [error X 1]../wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.19.0-dev6:3:54595] Cannot read properties of undefined (reading &#039;activate&#039;) 



== Elementor - Compatibility Tag ==
	
	CocoBasic - Neresa Elementor Widgets: Compatibility not specified
	Elementor Header & Footer Builder: Incompatible

````

</details>

### Agreement

- [X] I agree that my issue may be closed without action if it doesn't meet all the requirements.

**Comment by jddrums** [comment]
Thanks rodlopes79, is that a global setting, just concerned what else that might effect.

**Comment by nicholaszein** [comment]
Hello @jddrums! 

Thank you for submitting your issue! 🙏 

We are aware of this issue and our team is already working on a fix. We can’t say exactly when it'll be sorted out, but we kindly ask for your patience. Thanks for understanding! 😉

As a temporary workaround, you can set the "Inline Font Icons" feature in your WordPress **`Dashboard`** > **`Elementor`** > **`Settings`** > **`Features`** page to **`Inactive`**.

This feature replaces font icons with Inline SVGs and might help with performance.

Alternatively, you can add this custom CSS globally:

```css
.elementor-widget-n-accordion .e-n-accordion-item-title-icon span > svg {
  height: var(--n-accordion-icon-size);
  width: auto;
}
```

But if you don't know how to do it, it's better to simply deactivate the feature as explained above.

Best regards

**Comment by nicholaszein** [comment]
Hello there @jddrums!

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised was resolved in Elementor v3.19.1! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
